apps/web/src/components/workflow/preview/ErrorPrettyRender.tsx

```diff
@@ -1,37 +1,74 @@
 import { css } from '@novu/novui/css';
-import { Text } from '@novu/novui';
-import { Center } from '@novu/novui/jsx';
-import { IconErrorOutline } from '@novu/novui/icons';
+import { Button, Text, Title } from '@novu/novui';
+import { Center, Stack, VStack } from '@novu/novui/jsx';
+import { IconErrorOutline, IconExpandLess, IconExpandMore } from '@novu/novui/icons';
+import { useDisclosure } from '@mantine/hooks';
+import { hstack } from '@novu/novui/patterns';
+
+export function ErrorPrettyRender({ error: unparsedError }) {
+  const [isExpanded, { toggle }] = useDisclosure();
+  const error = 'response' in unparsedError ? unparsedError?.response?.data : unparsedError;
 
-export function ErrorPrettyRender({ error }) {
   return (
-    <div
+    <Stack
       className={css({
+        color: 'typography.text.feedback.alert',
         bg: 'input.border.error/20',
-        borderColor: 'input.border.error !important',
-        border: '1px solid !important',
+        border: 'solid',
+        borderColor: 'input.border.error',
         width: '100%',
-        borderRadius: 'input !important',
-        my: 'margins.layout.tabs.bottom !important',
+        borderRadius: 'input',
+        my: 'margins.layout.tabs.bottom',
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** 🧹😍‏